analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -48,7 +48,7 @@ import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.model.orEmpty
 import org.ossreviewtoolkit.model.toYaml
 import org.ossreviewtoolkit.model.utils.PackageCurationProvider
-import org.ossreviewtoolkit.model.utils.addPackageCurations
+import org.ossreviewtoolkit.model.utils.setPackageCurations
```

Comment:
Please also add to the commit message body something like this:

> The term "add" was misleading for these functions as data is actually overwritten instead of added. So use the term "set" instead to clarify.

Reply:
Also strictly speaking, this is again changing public API, so a breaking change, though I'd personally be fine with regarding `OrtResultExtensions` not to be part of the "primary" public API as these are "extensions" only, so I'll leave it to you if you want to mark this as breaking or not.